components/notes/CreateProspectNote.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react'
+import { Box, Button, TextField } from '@mui/material'
+import CancelIcon from '@mui/icons-material/Cancel'
+import CreateIcon from '@mui/icons-material/Create'
+import SaveIcon from '@mui/icons-material/Save'
+
+function CreateProspectNote({ prospect }) {
+  const [prospectNote, setProspectNote] = useState('')
+  const [isEditing, setIsEditing] = useState(false)
+
+  const handleProspectNoteChange = (event) => {
+    setProspectNote(event.target.value)
+  }
+
+  const handleCancelNote = () => {
+    setIsEditing(false)
+    setProspectNote('')
+  }
+
+  return (
+    <>
+      <Box style={{
+        display: 'flex',
+        justifyContent: 'flex-end',
+        alignItems: 'center'
+      }}>
+        {!isEditing && (
+          <Button style={{
+            textTransform: 'none',
+            fontSize: '11px',
```

Comment:
Let's use the appropiate measure for each type of style:
https://stackoverflow.com/questions/2915508/is-it-bad-to-work-with-pixels-in-css
Also, we might want to have a global buttons.css file where we set the same style for all buttons